code/game/machinery/machinery.dm

```diff
@@ -547,6 +547,9 @@ Class Procs:
 		if(check_arrest && R && (R.fields["criminal"] == "*Arrest*"))
 			threatcount += 4
 
+		if(check_arrest && R && (R.fields["criminal"] == "*Execute*" ))

```

Comment:
You can combine this line with 547 so you dint check check_arrest and R twice. Can even use a switch for that to improve performance but thats not required

Reply:
There they are now! Noticed by the way that these aren't actually beepsky's lines, but portable turret's behaviour is altered by these at portable_turret.dm. Probably one could just fix the turrets use beepsky's "judgebot" instead, but there they are for now